OCMF-de.md

```diff
@@ -633,6 +663,17 @@ Zuordnungen bis zu 40 Byte Länge vorgesehen.
+| LI   | Number         |0..1| Loss compensation Identification: This parameter is optional. A meter can use this value for adding a traceability ID number for justifying cable Loss characteristics from an Look up table specified in meter's documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| LR   | Number         |1..1| Loss compensation cable Resistance: This parameter is mandatory. A meter shall use this value for specifying the Cable Resistance Value used in cable Loss compensation computation. The unit of this value shall be specified in Meter's documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| LU   | Number         |0..1| Loss compensation cable resistance Unit: This parameter is optional. A meter can use this value for specifying the Unit of Cable Resistance Value given by LR field used in cable loss compensation computation. The unit of this value can be traced in OCMF format in addition to Meter's documentation.   
```

Comment:
LU is no number. 
As discussed it should be mandatory

Reply:
A comment on lines 671/672/673/674: There is reads "A meter can use this value..." -- I would like this to change (in all lines) to "a meter or a measuring capsule can use this value...", because it is obviously not necessarily the meter which has to perform the cable loss compensation calculations, but that can very well be an additional controller next to it.

Reply:
Maybe it is better to have such a clarifying comment before and stick to use "meter" only... just to keep the wording simple.